python/tank/commands/setup_project_wizard.py

```diff
@@ -393,7 +393,7 @@ def get_default_configuration_location(self):
 
         { "darwin": "/foo/bar/project_name",
           "linux2": None,
-          "win32" : "c:\foo\bar\project_name"}
+          "win32" : "c:\\foo\\bar\\project_name"}
```

Comment:
You are totally right. My bad! I'm not sure to using raw strings. I guess I'm used to my muscle memory of typing in `\\` instead of using `r""`, since I've started my life as a C++ programmer. If it's more Pythonic to use `r` strings, let's do it. I wonder what would happen in Python 3.6+ however if you wanted to use both `r` and `f` at the same time.

Reply:
Curiosity got the best of me. I tried it. You're allowed to have multiple prefixes, like `fr`/`rf` in front of a string.